### PR TITLE
add detail to README. Minor changes for consistency

### PR DESCRIPTION
**Pull request by zeuslawyer**
- change var naming for consistency
- add response data shape to README

### CODE REVIEW

File: package-lock.json
Comment (KuphJr) [inline package-lock.json:1]: Why is package-lock.json changing? Can you remove this from the PR?

File: yarn.lock
Comment (KuphJr) [inline yarn.lock:1]: Same here. This PR should not change yarn.lock.

**Review by KuphJr — CHANGES_REQUESTED** [review]
the .lock files shouldn't change.

**Comment by zeuslawyer** [comment]
Discarded.  Resubmitting in PR42

**Pull request closed without merging**